tests/test_types.py

```diff
@@ -2002,3 +2002,40 @@ def test_dumping_passes_context_to_inner_type_dump(self):
         context = object()
         DumpOnly(inner_type).dump('foo', context)
         assert inner_type.dump_context == context
+
+
+class TestTransform:
+    def test_loading_calls_inner_type_load_with_result_of_pre_load(self):
+        inner_type = SpyType()
+        transform = Transform(inner_type, pre_load=lambda foo: 'bar')
+        transformed = transform.load('foo')
+        assert inner_type.loaded == 'bar'
+
+    def test_post_load_calls_value_from_inner_type_load(self):
```

Comment:
The test name does not make sense to me. Why post load should call something (since it `should be called`).
`test_loading_calls_post_load_with_result_of_inner_type_load`